Resources/doc/getting_started/creating_an_admin.rst

```diff
@@ -0,0 +1,206 @@
+        {
+            $this->category = $category;
+        }
+
+        public function getCategory()
+        {
+            return $this->category;
+        }
+
+        // ...
+    }
+
+.. code-block:: php
+
+    // src/AppBundle/Entity/Category.php
+
+    // ...
+    use Doctrine\Common\Collections\ArrayCollection;
+    // ...
+
+    class Category
+    {
+        // ...
+
+        /**
+        * @ORM\OneToMany(targetEntity="BlogPost", mappedBy="category")
+        */
+        private $blogPosts;
+
+        public function __construct()
+        {
+            $this->blogPosts = new ArrayCollection();
+        }
+
+        public function getBlogPosts()
+        {
+            return $this->blogPosts;
+        }
+
+        // ...
+    }
+
+After this, create the schema for these entities:
+
+.. code-block:: bash
+
+    $ php app/console doctrine:schema:create
+
+.. note::
+
+    This article assumes you have basic knowledge of the Doctrine2 ORM and
+    you've set up a database correctly.
+
+Step 1: Create an Admin Class
+-----------------------------
+
+SonataAdminBundle helps you manage your data using a graphical interface that
+will let you create, update or search your model instances. The bundle relies
+on Admin classes to know which models will be managed and how these actions
+will look like.
+
+An Admin class decides which fields to show on a listing, which fields are used
+to search and how the create form will look like. Each model will have its own
+Admin class.
+
+Knowing this, let's create an Admin class for the ``Category`` entity. The
+easiest way to do this is by extending ``Sonata\AdminBundle\Admin\Admin``.
+
+.. code-block:: php
+
+    // src/AppBundle/Admin/CategoryAdmin.php
+    namespace AppBundle\Admin;
+
+    use Sonata\AdminBundle\Admin\Admin;
+    use Sonata\AdminBundle\Datagrid\ListMapper;
+    use Sonata\AdminBundle\Datagrid\DatagridMapper;
+    use Sonata\AdminBundle\Form\FormMapper;
+
+    class CategoryAdmin extends Admin
+    {
+        protected function configureFormFields(FormMapper $formMapper)
+        {
+            $formMapper->add('name', 'text');
+        }
+
+        protected function configureDatagridFilters(DatagridMapper $datagridMapper)
+        {
+            $datagridMapper->add('name');
+        }
+
+        protected function configureListFields(ListMapper $listMapper)
+        {
+            $listMapper->addIdentifier('name');
+        }
+    }
+
+So, what does this code do?
+
+* **Line 11-14**: These lines configure which fields are display on the edit
+  and create actions. The ``FormMapper`` behaves similar to the ``FormBuilder``
+  of the Symfony Form component;
+* **Line 16-19**: This method configures the filters, used to filter and sort
+  the list of models;
+* **Line 21-24**: Here you specify which fields are shown when all models are
+  listed (the ``addIdentifier()`` method means that this field will link to the
+  show/edit page of this particular model).
```

Comment:
addIdentifer() is not mandatory, i think we should add the action buttons here for show, edit and delete